File: java/djjavawork/HelloJava05/src/studyjava04_extends/Student.java
package studyjava04_extends;

public class Student {
	private String name;
	private int age;
	//Getter, Setter
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getAge() {
		return age;
	}
	public void setAge(int age) {
		this.age = age;
	}
	//생성자
	public Student(String name, int age) {
		this.name = name;
		this.age = age;
	}
	
	public void sleep() {
		System.out.println(getName()+"학생은 잡니다.");
	}
	
	public void study() {
		System.out.println(getAge()+"살 답게 공부합니다!");
	}
	

}

File: djCSharp/HelloCSharp03_Class/HelloCSharp03_01_start_class/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HelloCSharp03_01_start_class
{
    public class Student
    {
        //속성
        public int age; //인스턴스 변수
        public string name;
        //c#에서는 getter, setter가 있을까?
        public string hakbeon { get; set; }//dataGridView에선 필수!

        private string hp;
        public string phoneNumber
        {
            get { return hp; }
            set { hp = value; }
        }
    }
}
